<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Windscribe VPN</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/resources/icons/VPNGIF.gif</normaloff>:/images/resources/icons/VPNGIF.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="informLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>380</width>
      <height>87</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="accountStatusButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Check account status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>381</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAccountStatus">
   <property name="text">
    <string>checkStat</string>
   </property>
  </action>
  <action name="actionShowLoginDialog">
   <property name="text">
    <string>conDiscon</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>accountStatusButton</sender>
   <signal>clicked()</signal>
   <receiver>actionAccountStatus</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainButton</sender>
   <signal>clicked()</signal>
   <receiver>actionShowLoginDialog</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logoutButton</sender>
   <signal>clicked()</signal>
   <receiver>actionLogout</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainButton</sender>
   <signal>clicked()</signal>
   <receiver>actionConnect</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainButton</sender>
   <signal>clicked()</signal>
   <receiver>actionDisconnect</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
